mozilla-release/mobile/android/app/src/photon/res/values/styles.xml

```diff
@@ -1108,6 +1107,22 @@
         <item name="android:textSize">@dimen/cliqz_intro_content_size</item>
         <item name="android:buttonTint">@color/cliqz_intro_text_color</item>
     </style>
+
+    <style name="bond_dashboard_controls_title">
```

Comment:
move this to bond/styles.xml

Reply:
can't, because it's used inside photon/gecko_app.xml.
move it to bond means that I should have same style inside standard, but we don't have controls View at all inside standard.